docs/getting-started.md

```diff
@@ -74,13 +76,5 @@ docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$http_pr
 
 ## Using Promenade Behind a Proxy
 
-To use Promenade from behind a proxy, simply export `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables on the vagrant host prior to executing the `genesis.sh` and `join.sh` scripts respectively.  Alternatively, you may also export the `DOCKER_HTTP_PROXY`, `DOCKER_HTTPS_PROXY`, and `DOCKER_NO_PROXY` directly. Ensure you are running the script with `sudo -E` option to preserve the environment variables.
-
-```bash
-vagrant ssh n0
-cd /vagrant
-export DOCKER_HTTP_PROXY="http://proxy.server.com:8080"
-export DOCKER_HTTPS_PROXY="https://proxy.server.com:8080"
-export DOCKER_NO_PROXY="localhost,127.0.0.1"
-sudo -E /vagrant/up.sh /vagrant/configs/n0.yaml
-```
+To use Promenade from behind a proxy, set the `http_proxy`, `https_proxy`, and
+`no_proxy` variables in the input configuration fed to the `generate` command.
```

Comment:
This had been specified elsewhere in the docs, but it wasn't particularly well-written.

I consolidated those instructions into the configuration doc and refer to them from getting started now.